test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -200,4 +205,331 @@ Describe "TabCompletion" -Tags CI {
         $completionText = $res.CompletionMatches.CompletionText | Sort-Object
         $completionText -join ' '| Should Be 'blg csv tsv'
     }
+
+    Context "File name completion" {
+        BeforeAll {
+            $tempDir = Join-Path -Path $TestDrive -ChildPath "baseDir"
+            $oneSubDir = Join-Path -Path $tempDir -ChildPath "oneSubDir"
+            $oneSubDirPrime = Join-Path -Path $tempDir -ChildPath "prime"
+            $twoSubDir = Join-Path -Path $oneSubDir -ChildPath "twoSubDir"
+            $separator = [System.IO.Path]::DirectorySeparatorChar
```

Comment:
I see tests for `/`, but you should have a single test where the path contains both `\` and `/` and is normalized to the OS seperator